Library/Homebrew/cask/audit.rb

```diff
@@ -545,6 +545,37 @@ def check_livecheck_version
       false
     end
 
+    def check_livecheck_min_os
+      return unless cask.livecheckable?
+      return unless cask.livecheck.strategy == :sparkle
```

Comment:
```suggestion
    def check_livecheck_min_os
      return unless appcast?
      return unless cask.livecheckable?
      return unless cask.livecheck.strategy == :sparkle
```

Maybe?

Or at least `online?`, but `appcast?` seems consistent with `check_livecheck_version`.

Reply:
went for `online?` because it really doesn't have anything to do with the appcast here.